charts/kyverno/templates/deployment.yaml

```diff
@@ -71,14 +71,9 @@ spec:
           {{- with .Values.initResources }}
           resources: {{ tpl (toYaml .) $ | nindent 12 }}
           {{- end }}
-          securityContext:
-            runAsNonRoot: true
-            privileged: false
-            allowPrivilegeEscalation: false
-            readOnlyRootFilesystem: true
-            capabilities:
-              drop:
-                - ALL
+          {{- if .Values.securityContext }}
```

Comment:
I think might be better to do `with (include "kyverno.securityContext" .)` then do `toYaml . | indent 12`.

Reply:
Can a template _return_ something else that text ?